client/components/settings/alerts.js

```diff
@@ -65,13 +65,14 @@ export default class Alerts extends React.Component {
                   sendIfText={ this.props.sendIfText }
                 />
 
-                <div className="panel-body">
+                <div className="table-responsive">
```

Comment:
No, it must be applied to an element wrapping the table ([Bootstrap reference](http://getbootstrap.com/css/#tables-responsive)).